source/_integrations/alarm_control_panel.xiaomi_miio.markdown

```diff
@@ -1,38 +0,0 @@
----
```

Comment:
Alarm control panel was is the gateway integration, however at the start I only wrote the code to control the alarm of the gateway and was not thinking I was going to add more functionality. But due to requests from users I ended up writing loads of more code and now it can do a lot more stuff than only the alarm.

So the documentation is not removed, it is just put together with the gateway documentation as one of the features you get when setting up the gateway platform through config flow.